<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iGPMT</class>
 <widget class="QMainWindow" name="iGPMT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2,3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="Heading" name="accountsHead">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Accounts</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DetailsContainer" name="accountsCont">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item alignment="Qt::AlignHCenter">
             <widget class="ConfirmButton" name="add_account">
              <property name="text">
               <string>ADD ACC.</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="RejectButton" name="remove_account">
              <property name="text">
               <string>REMOVE ACC.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="Heading" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Tasks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TasksContainer" name="tasksCont"/>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="ConfirmButton" name="add_jobs_butt">
            <property name="text">
             <string>ADD TO JOBS</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
          <item alignment="Qt::AlignHCenter">
           <widget class="Heading" name="jobsHead">
            <property name="text">
             <string>Planned Jobs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="JobsContainer" name="jobsCont"/>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="ConfirmButton" name="perform_butt">
            <property name="text">
             <string>PERFORM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="ConfirmButton" name="refreshButton">
          <property name="text">
           <string>REFRESH</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Heading</class>
   <extends>QLabel</extends>
   <header>custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>ConfirmButton</class>
   <extends>QPushButton</extends>
   <header>custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>DetailsContainer</class>
   <extends>QScrollArea</extends>
   <header>custom_widgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TasksContainer</class>
   <extends>QScrollArea</extends>
   <header>custom_widgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JobsContainer</class>
   <extends>QScrollArea</extends>
   <header>custom_widgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RejectButton</class>
   <extends>QPushButton</extends>
   <header>custom_widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
